connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -57,6 +57,10 @@
     public static final String VALUE_CONVERTER_CLASS_DOC =
             "Converter class for value Connect data that implements the <code>Converter</code> interface.";
 
+    public static final String VALUE_CONVERTER_SCHEMA_CACHE_SIZE_CONFIG = "value.converter.schemas.cache.size";
```

Comment:
Okay I can change that. I was hoping to avoid magic properties that don't appear anywhere in the docs and isn't clear what the props should be.